webapp/src/telemetry/telemetryClient.ts

```diff
@@ -50,6 +50,7 @@ export const TelemetryActions = {
     LimitCardLimitReached: 'limit_cardLimitReached',
     LimitCardLimitLinkOpen: 'limit_cardLimitLinkOpen',
     VersionMoreInfo: 'version_more_info',
+    ClickChannelsRHSBoard: 'click_channels_RHS_board',
```

Comment:
see https://docs.google.com/document/d/1CJeMFxnwUq8njKd3S3zxnfqrwUz8zNDsTmUAYqgy_uo/edit#heading=h.gmxxmyi0tggz for event naming recommendations - sometimes useful to add "result of action" if one would have several actions on the same object.

Reply:
@dontoisme do you have any suggestion for the event name?

Reply:
@marianunez @esethna this is the telemetry we want to add for every time a user clicks a board card in the RHS panel (aka boards plugin). Do you have any suggestions for the name or the extra data information we should capture here?

Reply:
I am not sure if there is always a clear result of action that can be added to the telemetry as the action could be a UI interaction that can cause several effects (and could change with the iteration of the product, i.e. in this case the click of the board in the RHS in channels opens the board in Boards but in the future can open some other view in the RHS)

I would suggest a slight variation on this just to make it more readable but don't have a good result of action to suggest: 
```suggestion
    ClickChannelsRHSBoard: 'click_board_in_channels_RHS',
```